File: XPlaneParser/src/main/java/ms/xplaneparser/entity/Runway.java
package ms.xplaneparser.entity;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class Runway {
	public double startLat;
	public double startLon;
	public double endLat;
	public double endLon;
	public double length;
	public String runwayId;
	public SurfaceTypes surfaceType;


	@Override
	public String toString() {
		return "--Runway ID:  " + runwayId + " Length: " + length + " Surface: " + surfaceType;
	}

	public String getName() {
		return runwayId;
	}
}

File: XPlaneParser/src/main/java/ms/xplaneparser/entity/Runway.kt
package ms.xplaneparser.entity

class Runway(var startLat: Double, var startLon: Double, var endLat: Double, var endLon: Double, var length: Double, var name: String, var surfaceType: SurfaceTypes) {
    override fun toString(): String {
        return "--Runway ID:  $name Length: $length Surface: $surfaceType"
    }

}